authors/apps/authentication/views.py

```diff
@@ -119,3 +120,70 @@ def get(self, request):
             {"message": "Your Email has been verified,you can now login"}, 
             status.HTTP_200_OK
         )
+
+
+class ResetPasswordView(APIView):
+    def post(self, request):
+        try:
+            email = request.data["email"]
+            user = User.objects.filter(email=request.data["email"]).first()
+            if not user:
+                return Response({
```

Comment:
To protect our users, this check should not be included. Rather, a simple message should be returned of an email being sent. In this way, one will not know which emails we possess in our application when one is not a user of the app